core/src/main/scala/scalacss/Values.scala

```diff
@@ -75,6 +77,22 @@ object Literal extends TypedLiteralAliases {
     object ridge  extends Literal("ridge")  with BrStyle
     object inset  extends Literal("inset")  with BrStyle
     object outset extends Literal("outset") with BrStyle
+
+    // <timing-function>
+    sealed trait TimingFunctionDirection { def value: Value }
+    object start extends Literal("start") with TimingFunctionDirection
+    object end extends Literal("end") with TimingFunctionDirection
+
+    sealed trait TimingFunction
+    class CubicBezier(x1: Double, y1: Double, x2: Double, y2: Double) extends Literal(s"cubic-bezier($x1, $y1, $x2, $y2)") with TimingFunction
+    class Steps(steps: Int, direction: TimingFunctionDirection) extends Literal(s"steps($steps, ${direction.value})") with TimingFunction
+    object linear extends Literal("linear") with TimingFunction
+    object ease extends Literal("ease") with TimingFunction
+    object easeIn extends Literal("ease-in") with TimingFunction
+    object easeInOut extends Literal("ease-in-out") with TimingFunction
+    object easeOut extends Literal("ease-out") with TimingFunction
+    object stepStart extends Literal("step-start") with TimingFunction
+    object stepEnd extends Literal("step-end") with TimingFunction
```

Comment:
Also you'll need to add links to all of these + `start` + `end` to the `TypedLiteralAliases` trait at the top of this file.